web/chapter_views.py

```
@@ -0,0 +1,652 @@
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.models import User
+from django.db import models
+from django.http import HttpRequest, HttpResponse, HttpResponseRedirect
```

Ruff (0.8.2)

`django.http.HttpResponseRedirect` imported but unused

Remove unused import: `django.http.HttpResponseRedirect`

(F401)